app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Total Cost: <%=@recipe.total_cost %>  </h1>
+
+<h1><%=@recipe.name%></h1>
+<p><%=@recipe.complexity%></p>
+<p><%=@recipe.genre%></p>
+<h2>Ingredients</h2>
+
+<p> <% @recipe.ingredients.order(:name).each do |ingredient| %>
```

Comment:
`.order` is an AR method that queries the database, which we don't want to do in the view. Per MVC, we should encapsulate that in the model. Consider making a model method that orders all the ingredients that you could call on `@recipe` in the view